tests/simpledsn1/__snapshots__/basic-dsn-snapshot.snap.svg

```diff
@@ -1,11 +1,11 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="800" height="600"><style>
```

Comment:
something got messed up in this snapshot

Reply:
if you look at the image on github, it seems like there's probably a really crazy coordinate or something, i would just look at the soup and see if you can avoid regressing the snapshot

Reply:
I wonder if `mm` lib is not working correctly here

Reply:
I wonder if `mm` lib is not working as expected

Reply:
Yea, when I divide values by 1000, I get different results